spec/functional/resource/user/useradd_spec.rb

```diff
@@ -645,6 +645,15 @@ def lock_user_account
             expect(@error).to be_a(Mixlib::ShellOut::ShellCommandFailed)
             expect(@error.message).to include("Cannot unlock the password")
           end
+        elsif %w{rhel}.include?(OHAI_SYSTEM["platform_family"]) &&
+              OHAI_SYSTEM["platform_version"].to_f = 6.8
```

Comment:
Shouldn't this condition be based on platform family and architecture (`s390x` in this case)?